src/metrics/cognitive.rs

```diff
@@ -477,6 +484,54 @@ mod tests {
         );
     }
 
+    #[test]
+    fn python_expression_statement() {
+        // Boolean expressions containing `And` and `Or` operators were not
+        // considered in assignments
+        check_metrics!(
+            "def f(a, b):
+                c = True and True",
+            "foo.py",
+            PythonParser,
+            cognitive,
+            [(cognitive, 1, usize)],
+            [(cognitive_average, 1.0)]
+        );
+    }
+
+    #[test]
+    fn python_tuple() {
+        // Boolean expressions containing `And` and `Or` operators were not
+        // considered inside tuples
+        check_metrics!(
+            "def f(a, b):
+                return \"%s%s\" % (a and \"Get\" or \"Set\", b)",
+            "foo.py",
+            PythonParser,
+            cognitive,
+            [(cognitive, 2, usize)],
+            [(cognitive_average, 2.0)]
+        );
+    }
+
+    #[test]
+    fn python_elif_function() {
+        // Boolean expressions containing `And` and `Or` operators were not
+        // considered in `elif` statements
+        check_metrics!(
+            "def f(a, b):
+                if a and b:  # +2 (+1 and)
+                   return 1
+                elif c and d: # +2 (+1 and)
```

Comment:
Can you also add an additional test with two `elif`s?

Reply:
Sure! :)